README.rst

```diff
@@ -240,9 +240,16 @@ Display options
 
 By default, all rows in the **Results** table will be expanded except those that have :code:`Passed`.
 
-This behavior can be customized with a query parameter: :code:`?collapsed=Passed,XFailed,Skipped`.
+This behavior can be customized either with a query parameter: :code:`?collapsed=Passed,XFailed,Skipped`
+or by setting the :code:`render_collapsed` in a configuration file (pytest.ini, setup.cfg, etc).
 
+.. code-block:: ini
 
+  [pytest]
+  render_collapsed = True
```

Comment:
Is there an extra character after `True` here?

Reply:
Damn, you're right. 🤔 

1. How did you see them?
2. Any idea where they come from? They weren't visible characters, just some weird ASCII(maybe?).

Reply:
No idea :(

Reply:
How did you notice/see them? I can't tell in the Github diff.

Reply:
Weird they don't show up anymore... 🤔

Reply:
I fixed them @nicoddemus :) (the second commit)

Reply:
Great, that's fixed then